google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolStressTest.java

```diff
@@ -139,6 +142,8 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
   private void setupSession(final Session session) {
     ReadContext mockContext = mock(ReadContext.class);
     final ResultSet mockResult = mock(ResultSet.class);
+    //
+    // when(session.asyncClose()).thenReturn(ApiFutures.immediateFuture(Empty.getDefaultInstance()));
```

Comment:
Why is this commented out?

Reply:
The commented code has now been removed. It was erroneously added in the first place and caused this test to fail sometimes. While searching for the reason, it was commented out and never removed.